Zadanie 12.4 Ożywiamy kanban!/js/Column.js

```diff
@@ -0,0 +1,55 @@
+function Column(id, name) {
+    var self = this;
+    this.name = name || 'No name given';
+    this.$element = createColumn();
+
+    function createColumn() {
+        var $column = $('<div>').addClass('column');
+        var $columnTitle = $('<h2>').addClass('column-title').text(self.name);
+        var $columnCardList = $('<ul>').addClass('column-card-list');
+        var $columnDelete = $('<button>').addClass('btn-delete').text('x');
+        var $columnAddCard = $('<button>').addClass('add-card').text('Add a card');
+
+        $columnDelete.click(function () {
+            self.removeColumn();
+        });
+        $columnAddCard.click(function () {
+						var cardName = prompt("Enter the name of the card");
+						event.preventDefault();
+						$.ajax({
+								url: baseUrl + '/card',
+								method: 'POST',
+								data: {
+								name: cardName,
+								bootcamp_kanban_column_id: self.id
+								},
+								success: function(response) {
+										var card = new Card(response.id, cardName);
+										self.createCard(card);
+								}
+						})
+        });
+        $column.append($columnTitle)
+            .append($columnDelete)
+            .append($columnAddCard)
+            .append($columnCardList);
+
+        return $column;
+    }
+}
+
+Column.prototype = {
+    addCard: function (card) {
```

Comment:
createCard

Reply:
ok, wszystko poprawnione, aczklowiek przy próbie wykasowania kolumny wyrzuca mi błąd:

https://scr.hu/P96w62

to jest problem z serwerem prawda?